src/ListContainer.hpp

```diff
@@ -39,6 +39,65 @@ struct ipl_rangestruct {
 }
 #endif
 
+// class for linking IPs to filter groups, complete with comparison operators
+// allowing standard C++ sort to work
+class ipmap
+{
+public:
+    ipmap(uint32_t a, String g)
+    {
+        addr = a;
+        group = g;
```

Comment:
Codacy found an issue: [Variable 'group' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238856)